apps/agentic-chat/src/tools/getAccount.ts

```diff
@@ -35,12 +45,20 @@ export const getAccount = async ({
   assetsStore: AssetsStore;
   portfolioStore: PortfolioStore;
 }): Promise<GetAccountResult> => {
+  const chainId = networkToChainIdMap[network];
+
+  if (!chainId) {
+    throw new Error(`Unsupported network: ${network}`);
+  }
+
   if (!address) {
     throw new Error('No account connected');
   }
 
   const baseUrl = import.meta.env[
-    `VITE_UNCHAINED_${network.toUpperCase()}_HTTP_URL`
+    `VITE_UNCHAINED_${network
+      .replace('bsc', 'bnbsmartchain')
```

Comment:
preference to unify this naming convention. `bnbsmartchain` imo

Reply:
Rationale is `bnbsmartchain` is the exception (unchained), not the rule (appKit, bebop), so allows us to keep a single `networkToChainIdMap` - just gave it a try right now (without duplicating between `searchTokens` and `getAccount`) and surely:

<img width="716" alt="image" src="https://github.com/user-attachments/assets/34eeb6c2-936e-40fb-ba7e-990381cfc4c6" />

Also better for the model to speak a single terminology and not get confused in-between tool calls either context-wise

Reply:
hmm ok, feels like model should understand both realistically, same as eth vs ethereum, but that's fine, not a huge deal by any means